sentry/entrypoint.sh

```diff
@@ -2,7 +2,12 @@
 set -e
 
 if [ "$(ls -A /usr/local/share/ca-certificates/)" ]; then
-  update-ca-certificates
+  # for backups, let's redirect the console output to avoid it being written to backup file
+  if [[ $SENTRY_BACKUP == true ]]; then
```

Comment:
Probably a good idea to keep the output regularly in case it errors out for some reason. For backups, we don't want this in the json file so 👋

Reply:
This will also need an update in the develop documentation as the command for backup will be
`docker-compose run --rm -T -e SENTRY_LOG_LEVEL=CRITICAL SENTRY_BACKUP=true web export > sentry/backup.json`

Reply:
develop PR:
https://github.com/getsentry/develop/pull/851

Reply:
Reasonable, better approach is to use something like this. Going to close this PR 
`docker-compose run -v $(pwd)/sentry:/sentry-data/backup  --rm -T -e SENTRY_LOG_LEVEL=CRITICAL web export /sentry-data/backup/backup.json`

writes directly to a file directly

Reply:
As for 2, it feels like adding another level of indirection for logging errors can be confusing for users